migrations/main.sql

```diff
@@ -633,3 +633,29 @@ CREATE VIEW user_request_booking_hostel_view AS
     ORDER BY
         u.university_id,
         s.user_id;
+
+
+DROP VIEW IF EXISTS user_document_list_view;
+CREATE VIEW user_document_list_view AS
+    SELECT
+        ur.university_id,
+        ur.date_created,
+        ud.name,
+        ud.user_document_id,
+        ur.user_id
+    FROM
+        user_document ud
```

Comment:
Don't understand the connection between user_request.service_id and user_document.user_document_id

Reply:
LEFT JOIN user_request ur ON
		ur.service_id = ud.user_document_id

Don't understand the connection between user_request.service_id and user_document.user_document_id

Reply:
@victor-nuzhniy u was right, Egor created view not correct, so i already fixed it